s3auth-relay/pom.xml

```diff
@@ -174,6 +174,11 @@
             <groupId>com.google.guava</groupId>
             <artifactId>guava</artifactId>
         </dependency>
+        <dependency>
+            <groupId>commons-net</groupId>
+            <artifactId>commons-net</artifactId>
+            <version>3.3</version>
```

Comment:
Strange, I just used your branch, removed the version in commons-net and the project build fine.